paimon-web-ui/src/views/playground/components/query/components/console/components/controls/index.tsx

```diff
@@ -0,0 +1,170 @@
+/* Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License. */
+
+import styles from './index.module.scss'
+import { TableChartOutlined, StopOutlined } from "@vicons/material"
+import { LineChartOutlined, ClockCircleOutlined, DownloadOutlined } from "@vicons/antd"
+import {Refresh} from "@vicons/tabler"
+import {Copy20Regular} from "@vicons/fluent"
+import { useConfigStore } from '@/store/config'
+
+export default defineComponent({
+  name: 'TableActionBar',
+  setup() {
+    const { t } = useLocaleHooks()
+
+    const configStore = useConfigStore();
+    const isDarkMode = computed(() => configStore.theme === 'dark');
+
+    const activeButton = ref('table');
+    const setActiveButton = (button: any) => {
+      activeButton.value = button;
+    };
+
+    return {
+      t,
+      activeButton,
+      setActiveButton,
+      isDarkMode
+    }
+  },
+  render() {
+    return (
+      <div class={styles.container}>
+       <n-space class={styles.left} item-style="display: flex; align-items: center;">
+         <n-popover trigger="hover" placement="bottom-start" show-arrow={false} style="padding: 0"
```

Comment:
Remember to format the indented space of the code before submitting

Reply:
done.